code/Common/ImporterRegistry.cpp

```diff
@@ -211,8 +211,11 @@ void GetImporterInstanceList(std::vector<BaseImporter *> &out) {
     // Some importers may be unimplemented or otherwise unsuitable for general use
     // in their current state. Devs can set ASSIMP_ENABLE_DEV_IMPORTERS in their
     // local environment to enable them, otherwise they're left out of the registry.
-    const char *envStr = std::getenv("ASSIMP_ENABLE_DEV_IMPORTERS");
-    bool devImportersEnabled = envStr && strcmp(envStr, "0");
+
+    // These lines were commented out by jere9309 to support building on winuwp, where std::getenv is not available
+    //const char *envStr = std::getenv("ASSIMP_ENABLE_DEV_IMPORTERS");
+    //bool devImportersEnabled = envStr && strcmp(envStr, "0");
+    bool devImportersEnabled = false;
```

Comment:
Should this be wrapped in a UWP define block so it doesn't impact Desktop?

Reply:
It could, yeah. We don't use any of the dev importers right now, but maybe we'd want to test one of them out in the future